Makefile.toml

```diff
@@ -19,7 +19,7 @@ wasm-bindgen --out-dir dist --no-typescript --target web target/wasm32-unknown-u
 
 [tasks.build-wasm-watch]
 run_task = { name = ["build-wasm"] }
-watch = { watch = ["homotopy-common/src", "homotopy-core/src", "homotopy-graphics/src", "homotopy-web/src"] }
+watch = { watch = ["homotopy-common/src", "homotopy-core/src", "homotopy-graphics/src", "homotopy-web/src", "homotopy-model/src", "homotopy-gl/src"] }
```

Comment:
Thanks for this @H-Yulong, apologies for having made structural changes to the project without updating the build pipeline.